common/util.go

```diff
@@ -404,3 +404,47 @@ func uniqueVolumeMounts(mounts []v1.VolumeMount) []v1.VolumeMount {
 	}
 	return rMounts
 }
+
+// AreSlicesEqual returns true if the two provided string slices are equal.
+func AreSlicesEqual(first []string, second []string) bool {
+	if len(first) == 0 && len(second) == 0 {
+		return true
+	}
+	if len(first) == 0 || len(second) == 0 {
+		return false
+	}
+
+	tmp := make(map[string]int)
+	for _, str := range first {
+		tmp[str] = 1
+	}
+
+	for _, str := range second {
+		v, ok := tmp[str]
+		if !ok || v == -1 {
+			tmp[str] = -1
```

Comment:
we can return false here.

Reply:
Actually I didn't write that logic, I only moved this function to a more general place (from github eventsource utils) so it can be reused in more places, so I didn't want to introduce any unnecessary changes there. But anyway - sure, its NP I can change that.